app/src/main/java/com/netcetera/skopjepulse/countryCitySelector/CountryCityViewModel.kt

```diff
@@ -62,7 +62,33 @@ class CountryCityViewModel(application: Application) : AndroidViewModel(applicat
     }
     val jsonSelectedCities = gson.toJson(selectedCityItems)
     editor.putString(Constants.SELECTED_CITIES, jsonSelectedCities)
-    editor.commit()
+    editor.apply()
   }
 
+  /**
+   * Returns the list of countries and cities that are currently not
+   * present in the sharedPreferences
+   */
+  fun getSelectableCities(): List<Any>{
```

Comment:
Using `Any` as type in type safe language is big no NO, should be used only in extreme cases when other solution is not available. I suppose that here it is used because `_countryCityList` can contain both `CountryItem` and `CityItem`.

Think that this should be fixed as soon as possible and proper types should be used.

Reply:
Fixed: [Implement CountryCityItem interface](https://github.com/pulse-eco/pulse-eco-android/pull/5/commits/756351a5ec3edb4736dfe19422cfa8ec49354dfd)